etl/PAIMS_PIMS/PAIMS_PIMS/sqldevtst.th.gov.bc.ca.conmgr

```diff
@@ -1,20 +0,0 @@
-<?xml version="1.0"?>
```

Comment:
This doesn't seem right. With dontsavesensitive you should be able to save these conmgr files because the password will be stripped out.

Reply:
These connection managers were used by the deleted ELT packages

Reply:
If that is the case, where are the conmgr files that should be in use? assuming the package/project was changed to dontsavesensitive that should create a change in the conmgr files that are still in use in the ETL.